content/posts/gamebosu/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "gamebosu"
+img: "gamebosu-logo.png"
+date: 2021-05-13
+summary: "🕹 A custom ruleset for osu!lazer that adds a playable gameboy."
+---
+
+# status
+
+- Status : Active development
+- Last Update : 2021.514.0
+- Can import and play without error : Yes
+
+# overview
+
+- Creator : [Game4all](https://github.com/Game4all)
+- Availability : Available Publicly
+- GitHub Repositories : [Game4all/gamebosu](https://github.com/Game4all/gamebosu)
+
+Gamebosu is a rulesets that can make osu! play gameboy. The ruleset is developed by Game4all.
```

Comment:
Gamebosu is a ruleset that emulates a gameboy, and allows for playing gameboy ROMs. The ruleset is developed by Game4All.